x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/mitre/index.tsx

```diff
@@ -208,7 +210,7 @@ export const AddMitreThreat = ({ dataTestSubj, field, idAria, isDisabled }: AddI
           {values.length - 1 !== index && <EuiSpacer size="s" />}
         </div>
       ))}
-      <MyAddItemButton onClick={addItem} isDisabled={isDisabled}>
+      <MyAddItemButton data-test-subj="addMitre" onClick={addItem} isDisabled={isDisabled}>
```

Comment:
I see a mix in our code base of data-test-subj using hypen separators such as `add-mitre` versus camel case. I don't know if there is a convention documented in the contributing guide but just pointing it out.

This looks fine to me though. (I'm not picky about it)